src/index.css

```diff
@@ -84,27 +84,29 @@ code {
   position: absolute;
   z-index: 1;
   bottom: 42px;
-  left: calc(50% - (3 / 2) * (32px + 2 * 15px));
+  left: calc(50%);
   background-color: #fafafa;
   border-radius: 6.8px;
   box-shadow: 0px 1.7px 10.2px rgba(139, 139, 139, 0.36);
   display: flex;
   flex-direction: row;
+  width: 111px;
```

Comment:
Generally it's better to set container heights with the sizes of their children or with padding and margin, I'd prefer not to have a hardcoded height and width for this